lib/BridgedRoom.js

```diff
@@ -261,7 +334,44 @@ BridgedRoom.prototype._handleSlackMessage = function(message, ghost) {
         };
         return ghost.sendMessage(roomID, message);
     }
-    else if (message.files) {
+    else if (subtype === "message_changed") {
+        var previous_message = substitutions.slackToMatrix(message.previous_message.text);
+        var new_message = substitutions.slackToMatrix(message.message.text);
+
+        // The substitutions might make the messages the same
+        if (previous_message === new_message) {
+            console.log("Ignoring edit message because messages are the same post-substitutions.");
```

Comment:
We've changed our logging methods over to a nicer way, please look at one of the other log statements.